sentry-core/src/main/java/io/sentry/core/transport/AsyncConnection.java

```diff
@@ -176,7 +176,8 @@ private void flush() {
             throw new IllegalStateException(message);
           }
         } catch (IOException e) {
-          eventCache.store(event);
+          //          eventCache.store(event); TODO: its already saving at first, do we still need
```

Comment:
@bruno-garcia I believe this should be gone as we already save at the beginning.